docker/driver.go

```diff
@@ -0,0 +1,99 @@
+package docker
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/bblfsh/performance"
```

Comment:
[minor, optional] I wouldn't bother grouping this separately from the others below. YMMV.

Reply:
in my previous pr I was asked to adhere this grouping, am I missing something?
https://reviewable.io/reviews/bblfsh/performance/6/util/storage/influxdb/influxdb.go#..r3